x-pack/plugins/security_solution/jest.config.dev.js

```diff
@@ -9,9 +9,9 @@ module.exports = {
   preset: '@kbn/test',
   rootDir: '../../../',
   projects: [
-    '<rootDir>/x-pack/plugins/security_solution/common/*/jest.config.js',
+    // '<rootDir>/x-pack/plugins/security_solution/common/*/jest.config.js',
     '<rootDir>/x-pack/plugins/security_solution/server/*/jest.config.js',
-    '<rootDir>/x-pack/plugins/security_solution/public/*/jest.config.js',
-    '<rootDir>/x-pack/plugins/security_solution/scripts/junit_transformer/*/jest.config.js',
+    // '<rootDir>/x-pack/plugins/security_solution/public/*/jest.config.js',
+    // '<rootDir>/x-pack/plugins/security_solution/scripts/junit_transformer/*/jest.config.js',
```

Comment:
why these lines commented?

Reply:
Not meaning to commit - will update. Thanks!